.github/ISSUE_TEMPLATE/docs_error.yaml

```diff
@@ -0,0 +1,31 @@
+name: API documentation error
+description: Report a typo or error found in API documentation
+labels: ["documentation"]
+body:
+  - type: textarea
+    attributes:
+      label: Description of the issue
+      description: Description of what issue you have discovered. Please include as to why you think the issue is an issue.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Current content
+      description: What does the specification currently say?
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Expected content
+      description: What do you think the specification should say instead?
```

Comment:
I'd rephrase _What do you think the specification should say instead?_ to _What should the specification say instead?_